### PR TITLE
Add task owner to dbt operator

### PR DESCRIPTION
**Pull request by wornjs**
Set dbt meta owner as task owner in all Cosmos `DbtxxxOperator`s.

Closes: #1065

### CODE REVIEW

**Comment by wornjs** [comment]
here is  the result in airflow web ui
스크린샷 2024-07-07 오후 11 18 59

스크린샷 2024-07-07 오후 11 19 24

File: cosmos/dbt/graph.py [hunk 1]
```diff
@@ -76,6 +76,10 @@ def name(self) -> str:
         """
         return self.resource_name.replace(".", "_")
 
+    @property
+    def owner(self) -> str:
+        return str(self.config.get("meta", {}).get("owner", "airflow"))
```
Comment (tatiana) [inline cosmos/dbt/graph.py:81]: An alternative to hard-code `airflow` could be to retrieve the default value from Airflow:
```
from airflow.configuration import conf

default_owner = conf.get("operators", "default_owner")
```

If you want to follow this path, please define `default_owner` in `settings.py`, together with other configurations.

**Review by tatiana — COMMENTED** [review]
This is looking great, @wornjs !

I left  a minor comment (sorry for doing this while the PR is in draft) - we can finish reviewing once the checks are passing and the PR is ready for review.

File: cosmos/dbt/graph.py (hunk 1 above)
Comment (wornjs) [reply cosmos/dbt/graph.py:81]: I'll check it out tonight

Comment (wornjs) [reply cosmos/dbt/graph.py]: if there is no owner in dbt config 
It would be better to specify the owner for the Airflow DAG.

**Comment by wornjs** [comment]
@tatiana 
plz check this

File: cosmos/airflow/graph.py [hunk 2]
```diff
@@ -106,14 +106,20 @@ def create_test_task_metadata(
...
+        task_owner = node.owner
 
     elif render_config is not None:  # TestBehavior.AFTER_ALL
         task_args["select"] = render_config.select
         task_args["selector"] = render_config.selector
         task_args["exclude"] = render_config.exclude
+        task_owner = ""
+
+    else:
+        task_owner = ""
```
Comment (tatiana) [inline cosmos/airflow/graph.py:118]: @wornjs thanks for addressing the feedback! Could you add a test to cover this use case as well? It is looking good and we'll release this as part of Cosmos 1.6.

File: tests/airflow/test_graph.py [hunk 3]
```diff
@@ -42,7 +42,7 @@
     depends_on=[parent_seed.unique_id],
     file_path=SAMPLE_PROJ_PATH / "gen2/models/parent.sql",
     tags=["has_child"],
-    config={"materialized": "view"},
+    config={"materialized": "view", "meta": {"owner": "parent_node"}},
```
Comment (wornjs) [inline tests/airflow/test_graph.py:45]: @tatiana 
here is the test code to check task owner

File: cosmos/dbt/graph.py (hunk 1 above)
Comment (tatiana) [reply cosmos/dbt/graph.py]: Thanks a lot, @wornjs !

File: cosmos/airflow/graph.py (hunk 2 above)
Comment (tatiana) [reply cosmos/airflow/graph.py]: I just refactored the code not to need this conditional, I believe coverage will be happy 🤞

Comment (tatiana) [reply cosmos/airflow/graph.py]: It worked! I'll rebase and merge, thanks for the contribution, @wornjs !

**Review by tatiana — APPROVED** [review]

**Pull request merged**